packages/core/workers/src/WorkerFarm.js

```diff
@@ -251,7 +268,13 @@ export default class WorkerFarm extends EventEmitter {
 
     worker.on('request', data => this.processRequest(data, worker));
 
-    worker.on('ready', () => this.processQueue());
+    worker.on('ready', () => {
+      this.readyWorkers++;
+      if (this.readyWorkers === this.options.maxConcurrentWorkers) {
+        this.emit('ready');
```

Comment:
I added a `ready` event to the WorkerFarm to show in the UI when the workers are ready (the slow bit is loading the JS/Wasm)